{
  "Header": {
    "Original": "CVE-2017-2592 (python-oslo-middleware before versions 3.8.1, 3.19.1, 3.23.1 is vulner ...)",
    "Line": 391996,
    "ID": "CVE-2017-2592",
    "Description": "(python-oslo-middleware before versions 3.8.1, 3.19.1, 3.23.1 is vulner ...)"
  },
  "Annotations": [
    {
      "Original": "- python-oslo.middleware 3.19.0-3 (bug #852742)",
      "Line": 391997,
      "Type": "package",
      "Package": "python-oslo.middleware",
      "Kind": "fixed",
      "Version": "3.19.0-3",
      "BugNo": 852742
    },
    {
      "Original": "NOTE: https://launchpad.net/bugs/1628031",
      "Line": 391998,
      "Type": "NOTE",
      "Description": "https://launchpad.net/bugs/1628031"
    }
  ]
}